worker/include/RTC/RTCP/CompoundPacket.hpp

```diff
@@ -63,7 +64,7 @@ namespace RTC
 			~CompoundPacket() = default;
 
 			friend struct std::default_delete<RTC::RTCP::CompoundPacket>;
```

Comment:
Do we still need `default_delete` and public `ReturnIntoPool` below?

Reply:
updated